src/mscorlib/src/System/Array.cs

```diff
@@ -976,7 +976,7 @@ public Object Clone()
             if (array==null)
                 ThrowHelper.ThrowArgumentNullException(ExceptionArgument.array);
             Contract.EndContractBlock();
-            return BinarySearch<T>(array, 0, array.Length, value, null);
+            return BinarySearch<T>(array, 0, array.Length, value, (IComparer<T>)null);
```

Comment:
@hughbe API review process is documented in [corefx repo](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/api-review-process.md). The discoverability of the docs is not ideal (should improve over December - if anyone wants to help/review, I'll be happy to involve them ;-)).

Another angle is that coreclr repo didn't go through the [triage process](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/issue-guide.md#triage-rules---simplified) yet, so we have issues like #2188 which are in the wrong place (coreclr instead of corefx) and incorrectly marked :( ... I hope we will get to it early next year. I can do a quick sweep now of "up for grabs" and "api-addition" in coreclr to spot the worst offenders ...

Sorry for the confusion.